qcog_python_client/qcog.py

```diff
@@ -112,6 +121,38 @@ def __init__(self) -> None:  # noqa: D107
         self._training_parameters: dict | None = None
         self._trained_model: dict | None = None
         self._inference_result: dict | None = None
+        self._loss: Matrix | None = None
+
+    @property
+    def loss(self) -> Matrix | None:
```

Comment:
Btw I just noticed that the output is the matrix. So maybe we should explain the shape and structure of this matrix in the docstring? what's along each axis etc